dolphinscheduler-worker/src/main/resources/application.yaml

```diff
@@ -89,3 +89,13 @@ management:
 
 metrics:
   enabled: true
+
+# Override by profile
+
+---
+spring:
+  config:
+    activate:
+      on-profile: mysql
+  datasource:
+    driver-class-name: com.mysql.jdbc.Driver
```

Comment:
Also add the postgresql part ?

```yaml
---
 spring:
   config:
     activate:
       on-profile: postgresql
   datasource:
     driver-class-name: org.postgresql.Driver
```

Reply:
Don't need, it because the other YAML file's default is PostgreSQL. Only the tools module needs to define Mysql && PostgreSQL part. Due [this](https://github.com/apache/dolphinscheduler/pull/9997/files#diff-eed87c5c23a714652182ddde16029973958176002fea3b59fb8ec4d359a33893R28)

Reply:
Oh right